boards/arm/mimxrt1060_evk/Kconfig.defconfig

```diff
@@ -0,0 +1,47 @@
+# Kconfig - MIMXRT1060-EVK board
+#
+# Copyright (c) 2018, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if BOARD_MIMXRT1060_EVK
+
+config BOARD
+	default "mimxrt1060_evk"
+
+if GPIO_MCUX_IGPIO
+
+config GPIO_MCUX_IGPIO_1
+	def_bool y
+
+config GPIO_MCUX_IGPIO_2
+	def_bool n
```

Comment:
Remove the configs with `def_bool n`; they are redundant.

Reply:
will do